src/pages/SignIn/SignIn.ts

```diff
@@ -38,28 +40,37 @@ export default class SignIn extends Component {
 
 	postRender() {
 
-		const form = this._element.firstChild as HTMLFormElement;
+		const form = this._element.querySelector("form");
 
 		if (form) {
 			form.addEventListener("submit", (e) => {
 				e.preventDefault();
 
 				const inputs = Array.from(form.querySelectorAll("input"));
-				inputs.forEach(
-					input => console.log(input.value)
-				);
 				const isValid = inputs.every(
 					// @ts-ignore
 					input => validate(input.type, input.value)
 				);
 
-				if (isValid)
-					replaceElementWithComponent("#root", new Chat());
+				if (true) {
```

Comment:
Тут наверное должно быть так?
```suggestion
				if (isValid) {
```